.circleci/workflows.yml

```diff
@@ -564,6 +564,7 @@ jobs:
                   PATH="venv/bin:$PATH" script/bqetl metadata update \
                     --sql-dir /tmp/workspace/generated-sql/sql/ \
                     /tmp/workspace/generated-sql/sql/
+                  PATH="venv/bin:$PATH" script/bqetl format /tmp/workspace/generated-sql/sql/
```

Comment:
Running the formatting as part of the SQL generation to ensure that identifiers are fully qualified when the content gets pushed to `generated-sql`.

Reply:
IIUC we could still have unqualified queries on `main`? Though they'll be qualified in `generated-sql`. Would be nice to have a validation check so we don't end up in that situation, maybe in a pre-commit?

Reply:
My understanding is that we need to run this after sql generation because some of the generated sql doesn't have fully qualified identifiers.  Everything in main should be caught by `verify-format-sql`